components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -590,6 +590,11 @@ sealed class DownloadAction : BrowserAction() {
      * Updates the provided [download] on the [BrowserState].
      */
     data class UpdateDownloadAction(val download: DownloadState) : DownloadAction()
+
+    /**
+     * Restore the [BrowserState.downloads] state from the storage.
+     */
+    object RestoreDownloadsState : DownloadAction()
```

Comment:
nit: Let's keep the `Action` postfix. Eventually, I think we can remove `Action` from all these names. I filed a ticket for that a while ago, but for now let's keep it consistent.